native/src/components/RemoteContent.tsx

```diff
@@ -181,7 +183,7 @@ const RemoteContent = ({
       // https://github.com/react-native-webview/react-native-webview/issues/1869
       setSupportMultipleWindows={false}
       style={{
-        height: webViewHeight,
+        height: visible ? webViewHeight + dimensions.ttsPlayerHeight : webViewHeight,
```

Comment:
:x: I sadly think that this is not the right place for this as the `lastUpdate` timestamp is still hidden behind the player. Probably you have to move it one level up to `Page`.